lib/kapnismology/terminal.rb

```diff
@@ -1,20 +1,29 @@
 module Kapnismology
   class Terminal
-
     def self.green(msg)
-      "\e[32m\e[1m#{msg}\e[0m"
+      colorize("\e[32m", msg)
     end
 
     def self.red(msg)
-      "\e[31m\e[1m#{msg}\e[0m"
+      colorize("\e[31m", msg)
     end
 
     def self.yellow(msg)
-      "\e[33m\e[1m#{msg}\e[0m"
+      colorize("\e[33m", msg)
     end
 
     def self.bold(msg)
-      "\e[1m#{msg}\e[0m"
+      colorize('', msg)
```

Comment:
which makes me think you might want to inverse the arguments in your colorize method:
msg first and an optional color second so in this case you'd just do `colorize(msg)`.
also it makes more sense IMHO, it's like saying "make this msg yellow" and not "make yellow this msg".